addon/components/frost-bunsen-cell/component.js

```diff
@@ -1,12 +1,11 @@
 import _ from 'lodash'
-import Ember from 'ember'
 import computed, {readOnly} from 'ember-computed-decorators'
 
-import PropTypesMixin, {PropTypes} from 'ember-frost-bunsen/mixins/prop-types'
+import FrostComponent, {PropTypes} from 'ember-frost-component'
```

Comment:
Now that I look at the implementation, I am a little concerned in general with this approach. 

We are putting in a big conceptual abstraction. We now have a `FrostComponent` instead of an `Ember.Component`, and while that provides us with certain benefits, it also creates a Big Dark Area of Uncertainty for developers. "What on Earth is a `FrostComponent`?" they will ask themselves. Sure, you know and maybe I know, but the average rank-and-file developer won't know, and this will be a source of magic and vague abstraction.

In general, there is always a balance to be struck between eliminating boilerplate code and doing large and potentially confusing abstractions, and I'm worried that this may be pushing too far in the wrong direction.